### PR TITLE
ixia-c licensed version's sample test (bgpv4 forwarding)

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: scripts/bgpv4_forwarding.go [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+	// fetch Bgpv4 metrics and wait for sessions to be up and routes count to be correct
+	mr := api.NewMetricsRequest()
+	mr.Bgpv4()
+	waitFor(
+		func() bool {
+			res, err := api.GetMetrics(mr)
+			checkResponse(res, err)
+
+			bgp1m := res.Bgpv4Metrics().Items()[0]
```
Comment (apratimmukherjee) [inline scripts/bgpv4_forwarding.go:55]: Maybe a better name here ? Could not follow what bgp1m and bgp2m refer to.

File: scripts/bgpv4_forwarding.go [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+	p2d1BGPv4PeerRoutes.Addresses().Add().
+		SetAddress("20.20.20.1").
+		SetPrefix(24).
+		SetCount(5).
+		SetStep(1)
+
+	// create a flow and set the endpoints
+	f1 := config.Flows().Add().SetName("p1.v4.p2")
+
+	f1.TxRx().Device().SetTxNames([]string{p1d1BGPv4PeerRoutes.Name()}).SetRxNames([]string{p2d1BGPv4PeerRoutes.Name()})
```
Comment (apratimmukherjee) [inline scripts/bgpv4_forwarding.go:182]: Fix tabbing ?

File: scripts/bgpv4_forwarding.go [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+	// enable per flow metrics tracking
+	f1.Metrics().SetEnable(true)
+	// set size, count and transmit rate for all packets in the flow
+	f1.Size().SetFixed(512)
+	f1.Rate().SetPps(500)
```
Comment (apratimmukherjee) [inline scripts/bgpv4_forwarding.go:188]: Maybe lower pps to 100 or so for lower end setups.

**Review by apratimmukherjee — APPROVED** [review]
Minor changes requested.

File: scripts/bgpv4_forwarding.go (hunk 1 above)
Comment (biplamal) [reply scripts/bgpv4_forwarding.go]: renaming done

File: scripts/bgpv4_forwarding.go (hunk 2 above)
Comment (biplamal) [reply scripts/bgpv4_forwarding.go:180]: Fixed

File: scripts/bgpv4_forwarding.go (hunk 3 above)
Comment (biplamal) [reply scripts/bgpv4_forwarding.go]: Done

**Review by apratimmukherjee — APPROVED** [review]
Looks good to me.

**Pull request merged**